test_environments/message_and_event_brokers/kafka/sasl_scram/with_tls/scram_sha_256/README.md

```diff
@@ -0,0 +1,26 @@
+This is a simple setup of Kafka with SASL_SCRAM (SHA-256 algorithm) authentication and TLS encryption.
```

Comment:
```suggestion
# Testing Kafka broker with SASL_SCRAM (SHA-256 algorithm) and TLS

This is a simple setup of Kafka with SASL_SCRAM (SHA-256 algorithm) authentication and TLS encryption.
```

Reply:
I added `# Setup Kafka broker with SASL SCRAM (SHA-256) authentication and TLS encryption`